tests.out

```diff
@@ -0,0 +1,101 @@
+Skipped 9 tests
+Testing '.[0]' at line number 37
+Hit the number of tests limit (1), breaking
+1 of 1 tests passed (0 malformed)
+==9681== 17 bytes in 1 blocks are definitely lost in loss record 4 of 45
```

Comment:
Are these expected leaks??

Reply:
On my version of the branch there were no leaks. Need to check.

Reply:
I think you don't use this file, `tests.out`, right (`git grep tests.out` produces nothing)?  So let's remove this file.

Reply:
AH right this file was accidentally committed. It was an intermediate support file I've generated during my hunt for leaks.

BTW it shows the usage of a side feature I've implemented which allows to -skip and -take a slice out of a test file. I've used it to bisect and find the leaking tests

Reply:
Yeah, I saw.  I liked it.

BTW, with the new `eval` stuff in #1843 we could replace the `src/jq_test.c` with jq-coded test-running utilities, and then `--run-tests` could even be removed.